### PR TITLE
Testing coverage

### PR DESCRIPTION
**Pull request by yobuir**
### What does this PR do?
This pull request adds unit tests to ensure that all routes in the application are functioning correctly. By doing so, we can catch any regressions that may occur as we continue to develop and maintain the application.

### Description of Task to be completed?
Ensure all Tests are Passing and Working as expected!

### How should this be manually tested?
To manually test  this, you can follow these steps:

`git clone https://github.com/atlp-rwanda/ecommerce-app-predators-bn.git`
```
run npm install
run npm run coverage
``` 

Screenshots (if appropriate)
N/A

### CODE REVIEW

**Review by ALVINdimpos — APPROVED** [review]

**Review by vanessa-abewe — APPROVED** [review]

**Review by Muhet — CHANGES_REQUESTED** [review]
Why the test on push are not passing? if it is possible can you fix it?

**Review by Muhet — DISMISSED** [review]
Why the test on push are not passing? if it is possible can you fix it?

**Review by faid-terence — APPROVED** [review]

**Review by NTElissa — APPROVED** [review]
good progress

File: .nyc_output/processinfo/index.json [hunk 1]
```diff
@@ -1 +1,5 @@
-{"processes":{"f1df5a1f-e139-46df-b7b9-49b2b36476b8":{"parent":null,"children":[]}},"files":{"C:\\Users\\Dr. Doreen Wa JJ\\Desktop\\Predators Ecommerce Backend\\ecommerce-app-predators-bn\\config\\test.cjs":["f1df5a1f-e139-46df-b7b9-49b2b36476b8"]},"externalIds":{}}
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (chanceDira) [inline .nyc_output/processinfo/index.json]: Can you solve this issue, I think you got it while doing rebase

File: tests/Admin.test.js [hunk 2]
```diff
@@ -5,8 +5,20 @@ import dotenv from 'dotenv';
 dotenv.config();
 const expect = chai.expect;
 chai.use(chaiHttp);
-// Admin Tests
+
 describe('ADMIN PRE-CONFIGURED CREDENTIAL SIGIN', function() {
+  // it('should return a JWT token on successful login', async () => {
```
Comment (chanceDira) [inline tests/Admin.test.js]: If we are not using this we can delete these codes

**Review by chanceDira — APPROVED** [review]
Nice work @yobuir just one thing

File: .nyc_output/processinfo/index.json (hunk 1 above)
Comment (yobuir) [reply .nyc_output/processinfo/index.json]: fixed

**Pull request merged**